packages/autocomplete-core/src/__tests__/getInputProps.test.ts

```diff
@@ -334,20 +336,22 @@ describe('getInputProps', () => {
 
       userEvent.type(inputElement, '');
 
-      expect(onStateChange).not.toHaveBeenLastCalledWith({
-        prevState: expect.anything(),
-        state: expect.objectContaining({
-          status: 'loading',
-        }),
-      });
-      expect(onStateChange).toHaveBeenLastCalledWith({
-        prevState: expect.anything(),
-        state: expect.objectContaining({
-          status: 'idle',
-          collections: [],
-          isOpen: false,
-        }),
-      });
+      expect(onStateChange).not.toHaveBeenLastCalledWith(
```

Comment:
As it's checking that it's last called with `status: idle`, is this test needed?

Reply:
I didn't change this test logic but you've got a point. This assertion is required but it should probably use `.not.toHaveBeenCalledWith`. I'll take care of that in another commit.